src/NServiceBus.Testing.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -252,32 +252,32 @@ namespace NServiceBus.Testing
         public System.Collections.Generic.IReadOnlyCollection<NServiceBus.Routing.RoutingStrategy> RoutingStrategies { get; set; }
     }
     [System.Diagnostics.DebuggerDisplay("TestableSaga for {typeof(TSaga).FullName}")]
-    public class TestableSaga<TSaga, TSagaEntity>
-        where TSaga : NServiceBus.Saga<TSagaEntity>
-        where TSagaEntity :  class, NServiceBus.IContainSagaData, new ()
+    public class TestableSaga<TSaga, TSagaData>
+        where TSaga : NServiceBus.Saga<TSagaData>
+        where TSagaData :  class, NServiceBus.IContainSagaData, new ()
     {
-        public TestableSaga(System.Func<TSaga> sagaFactory = null, System.DateTime? initialDateTime = default) { }
-        public System.DateTime CurrentTime { get; }
+        public TestableSaga(System.Func<TSaga> sagaFactory = null, System.DateTimeOffset? initialCurrentTime = default) { }
+        public System.DateTimeOffset CurrentTime { get; }
```

Comment:
> ...consider [DateTimeOffset](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset) as the default date and time type for application development.

_—[Choose between DateTime, DateTimeOffset, TimeSpan, and TimeZoneInfo](https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime#the-datetimeoffset-structure)_

Reply:
I did consider it, but there's no possible concept of multiple time zones here doesn't really exist. It can also only be mutated by AdvanceTime by using a TimeSpan.

Reply:
Out of interest, is there any reason to _prefer_ `DateTime` over `DateTimeOffset`?

Reply:
It ultimately doesn't even matter whether you're using UTC or local. It's just one timeline being managed completely internally. Adding the concept of there being a potential offset actually makes things more confusing.